docs/articles/rag-application-communication-system.md

```diff
@@ -0,0 +1,164 @@
+
+### 4.3 Purpose of fine-tuning: relieving data frictions
+A RAG system is made primarily for its users. Hence one of the biggest issues of using a generalist model like GPT-3.5/4 on a long term basis: you have an audience alignment gap. This is even more striking if you’re working in another language than English (like myself): ChatGPT feels immediately “alien” with weird twists of languages that are not to be found in standard French.
+
+Reshaping style is the most shocking impact of fine-tuning. A few months ago I released an extreme version of user-LLM communication: MonadGPT, an instruction model trained on 17th century English. What took me the longest was to come up with the most fitting communication scheme: train the models on a couple of answers from historical texts and generated questions in contemporary English, simply because I should absolutely not expect the users to mimic 17th century language. Users should not have to mimic the style of any document resources either. 
+
+Similarly to MonadGPT you should look for the following alignment design:
+* Generated queries should be as close as possible as the real queries sent by users, which means likely purposefully generating “bad” data. In my current instruction dataset, most queries have no punctuation, some are in all caps or missing words, with typos.
+* Generated answers have to convey the general “brand” style of your system. Do not hesitate to re-generate the answer, reformulate the existing one, or even rewrite them manually.
+
+Relatedly, you have to anticipate the “persona” of your model. Whether you like it or not, users of interactive systems will ask meta-referential questions: who are? What can you do? Many advanced LLM systems overlooked this aspect to their detriment (with new models trained from scratch erroneously stating they are ChatGPT, since this is the default AI persona in much of the training data these days). I recommend here to combine both prepared answers in the instruction dataset with a series of “biographical” documents about the model in the RAG dataset.
+
+To assess style and frictions, the best way may ultimately be to collect feedback from your users. You don’t need anything fancy: I would just recommend an open-ended comment section somewhere (and green/red buttons).
+
+# 5. Will RAG redefine LLMs?
+By mid-2024, we are probably at a crossroad. There's always the possibility of a major breakthrough in LLM research (Q*, Monte Carlo Tree Search, diffusion models, or state space models…) that could shift the focus back to model architecture. If not LLMs will be increasingly pushed to the backstage of larger, more complex systems.
+
```

Comment:
This is a pretty big step back from the original claim that RAG will eat LLMs.

Reply:
Yes, maybe this rephrasing: "While there's always the possibility of a major breakthrough in LLM research (Q*, Monte Carlo Tree Search, diffusion models, or state space models…) that would shift the focus back to model architecture, LLMs seems to be increasingly pushed to the backstage of larger, more complex systems."